DAO.sol

```diff
@@ -607,7 +607,9 @@ contract DAO is DAOInterface, Token, TokenSale {
     function newContract(address _newContract){
         if (msg.sender != address(this) || !allowedRecipients[_newContract]) return;
         // move all funds
-        _newContract.call.value(address(this).balance)();
+        if (!_newContract.call.value(address(this).balance)()) {
+            return;
```

Comment:
Should we use `throw` like is used [here](https://github.com/slockit/DAO/blob/master/DAO.sol#L510)